libcodechecker/server/server.py

```diff
@@ -949,6 +966,17 @@ def start_server(config_directory, package_data, port, config_sql_server,
                                      check_env,
                                      manager)
 
+    def signal_handler(*args, **kwargs):
+        """
+        Handle SIGTERM to stop the server running.
+        """
+        LOG.info("Received shutdown request.")
+        http_server.terminate()
+        LOG.info("WEB server is shut down.")
+        sys.exit(0)
+
+    signal.signal(signal.SIGTERM, signal_handler)
```

Comment:
Please handle SIGINT here too, we are sending SIGINT from the main caller process.